uw-frame-components/portal/features/services.js

```diff
@@ -236,7 +236,9 @@ define(['angular'], function(angular) {
               //check dates
               var today = Date.parse(new Date());
               var startDate = Date.parse(new Date(feature.goLiveYear, feature.goLiveMonth, feature.goLiveDay));
-              var expirationDate = feature.buckyAnnouncement.endDate;
+              var isoDate = feature.buckyAnnouncement.endDate;
```

Comment:
How messy would it be to support both epoch time and ISO date in `feature.buckyAnnouncement.endDate`? As in, maybe, if it's an integer, it's an epoch time, but if it's an ISO date, well, then it's an ISO date?

Asking for a couple reasons.

1. Would ease the upgrade path, not having to update `features.json` in the tiers in the same transaction with upgrading to a `uw-frame` version with this changeset.
2. Would ease the upgrade path for non-MyUW adopters of `angularJS-portal`.